File: src/main/java/lotto/config/AppConfig.java
package lotto.config;

import lotto.controller.LottoController;
import lotto.service.LottoService;
import lotto.view.InputView;
import lotto.view.OutputView;

public class AppConfig {

    public static LottoService createService() {
        return new LottoService();
    }

    public static LottoController createController() {
        return new LottoController(InputView.Companion.create(), OutputView.create(), createService());
    }
}

File: src/main/kotlin/lotto/config/AppConfig.kt
package lotto.config

import lotto.controller.LottoController
import lotto.service.LottoService
import lotto.view.InputView
import lotto.view.OutputView

object AppConfig {

    private fun createService(): LottoService =
        LottoService()

    fun createController(): LottoController =
        LottoController(
            InputView.create(),
            OutputView.create(),
            createService()
        )
}
